app/components/DisplayAddress.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import clsx from 'clsx';
+import { chunkString } from '~/utils/basicHelpers';
+
+interface Props {
+    address: string;
+    lineClassName?: string;
+    outerClassName?: string;
+}
+
+const lineLength = 10;
+
+export default function DisplayAddress({
+    address,
+    lineClassName,
+    outerClassName,
+}: Props) {
+    return (
+        <div className={clsx(outerClassName, 'textCenter')}>
```

Comment:
I think we should consider making this monospaced to make it easier to validate.
```
4H4QJaLRQv
p8mzcDCxVu
11scJCcrvt
Xnd1fhwjSw
HTDTHcXuD4
```